File: sample-chat-java/app/src/main/java/com/quickblox/sample/chat/java/ui/adapter/listeners/MessageLongClickListener.java
package com.quickblox.sample.chat.java.ui.adapter.listeners;

import android.view.View;

import com.quickblox.chat.model.QBChatMessage;

public interface MessageLongClickListener {
    void onMessageLongClicked (int itemViewType, View view, QBChatMessage qbChatMessage);
}

File: sample-chat-kotlin/app/src/main/java/com/quickblox/sample/chat/kotlin/ui/adapter/listeners/MessageLongClickListener.kt
package com.quickblox.sample.chat.kotlin.ui.adapter.listeners

import android.view.View
import com.quickblox.chat.model.QBChatMessage


interface MessageLongClickListener {
    fun onMessageLongClicked(itemViewType: Int?, view: View, qbChatMessage: QBChatMessage?)
}